media-video/vlc/vlc-3.0.4.recipe

```diff
@@ -154,8 +152,8 @@ BUILD_REQUIRES="
 	devel:libQt5Widgets$secondaryArchSuffix >= 5.7
 	devel:libsamplerate$secondaryArchSuffix
 #	devel:libschroedinger_1.0$secondaryArchSuffix # fix libschroedinger.pc
-	devel:libsdl_1.2$secondaryArchSuffix
-	devel:libsdl_image_1.2$secondaryArchSuffix
+	devel:libSDL2_2.0$secondaryArchSuffix
+	devel:libSDL2_image_2.0$secondaryArchSuffix
```

Comment:
Why SDL libs are needed? I thought VLC dropped SDL support in 3.x

Reply:
Removed.